gremlin-go/driver/request.go

```diff
@@ -75,3 +75,19 @@ func makeBasicAuthRequest(auth string) (req request) {
 		},
 	}
 }
+
+func makeCloseSessionRequest(client *Client) request {
+	return request{
+		requestID: uuid.New(),
+		op:        "close",
+		processor: "session",
+		args: map[string]interface{}{
+			"session": client.session,
+		},
+	}
+}
+
+func addSessionArgsToRequest(request *request, sessionId string) {
```

Comment:
Since this mutates the request which is not aligned with how the rest of the requests are created, why not add the ability to add the sessionId to `makeByteCode` and `makeStringRequest`